### DISCUSSION TITLE
Project Update and Tasks

### DISCUSSION THREAD

**Discussion by Joseph-Kassab** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#50] [Announcements]
### **Updates and TODO items**

1. SRS and Diagrams - Only 7 people have chosen a task. If you don't pick one soon, someone else will do it instead as we are approaching the deadline. Please visit the GitHub discussion titled "Task Assignment - SRS and Diagrams."

2. Wiki - Choose one of the 15 tasks on Sujana's GitHub discussion post titled "Task Assignment - Wiki." Make sure to follow the guidelines on Q&A discussion so it looks like this: https://github.com/Netflix/Hystrix/wiki

3. Jupyter documentation with a link in the README.

4. App not working on CSE server (check Q&A discussion).

5. Still exposing wrong port number (82 for web).

6. During Monday's meeting, we'll go through the whole repository and make sure it is ready to present.

Thank you!

**Comment by farheen-akhter-code** [comment]
Hi 
I am picking #4 and #5 from Todo items.

**Comment by Siyamul33** [comment]
completed both srs and wiki task , I am available to do more
